index.html

```diff
@@ -42,6 +42,182 @@ <h2>I'm a Software Developer</h2>
+                        personalized reads; no accounts 
+                        or sign-ups required. Has been 
+                        the industry's standard.</p>
+                    <div class="">
+                        <ul class="work_list flex">
+                            <li><button class="works_button">HTML</button></li>
+                            <li><button class="works_button">Bootstrap</button></li>
+                            <li><button class="works_button">Ruby</button></li>
+                        </ul>
+                    </div>
+                        <button class="see_project button_enabled">See project</button>
+                </div>
+
+                <div class="work">
+                    <h2>Profesional Art <br>Printing Data</h2>
+                    <p>A daily selection of privately 
+                        personalized reads; no accounts 
+                        or sign-ups required. Has been 
+                        the industry's standard.</p>
+                    <div class="">
+                        <ul class="work_list flex">
+                            <li><button class="works_button">HTML</button></li>
+                            <li><button class="works_button">Bootstrap</button></li>
+                            <li><button class="works_button">Ruby</button></li>
+                        </ul>
+                    </div>
+                        <button class="see_project button_enabled">See project</button>
+                </div>
+
+                <div class="work">
+                    <h2>Profesional Art <br>Printing Data</h2>
+                    <p>A daily selection of privately 
+                        personalized reads; no accounts 
+                        or sign-ups required. Has been 
+                        the industry's standard.</p>
+                    <ul class="work_list flex">
+                        <li><button class="works_button">HTML</button></li>
+                        <li><button class="works_button">Bootstrap</button></li>
+                        <li><button class="works_button">Ruby</button></li>
+                    </ul>
+                    <button class="see_project button_enabled">See project</button>
+                </div>
+            </div>
+
+        </section>
+
+        <section id="about" class="about">
+            <div class="about_me flex_column">
+                <h2>About me</h2>
+                <p>Hello I’m a software developer! 
+                    I can help you build a product , 
+                    feature or website Look through 
+                    some of my work and experience! 
+                    If you like what you see and have 
+                    a project you need coded, don’t 
+                    hesitate to contact me.</p>
+                <button class="button_enabled">Get my resume</button>
+                <img src="assets/Frame.png" alt="frame" class="frame">
```

Comment:
- [ ] kindly consider adding this as background image not an image as its a decoration and this will help you to full-fill the below requirement
- > The section has the proper background image (decorations are always backgrounded images, not `img` tags)